### PR TITLE
Get actions for orgs

### PR DESCRIPTION
**Pull request by davidlawrencer**
Quick PR to make organization info call available. Note I added a new property to actions so that they have the name of the organization that created them, and updated UI based on this new property.

Held off on integrating the call into UI because CellDataCoupler pod is still in the process of being removed.

### CODE REVIEW

File: FullCircle/FullCircle/Models/Action/Action.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ struct Action {
     let name: String
     let description: String
     let organizationID: String
+    let organizationName: String
```
Comment (msystang) [inline FullCircle/FullCircle/Models/Action/Action.swift:16]: Hey @davidlawrencer ! How is organizationName different from the name property?

Comment (davidlawrencer) [reply FullCircle/FullCircle/Models/Action/Action.swift:16]: great question! is `name` the org name? if so i will remove! @msystang

Comment (msystang) [reply FullCircle/FullCircle/Models/Action/Action.swift:16]: Yes! name is org name -- we can keep whichever property you feel is more intuitive! Right now I see organizationName has the actual org name on Firebase @davidlawrencer

Comment (msystang) [reply FullCircle/FullCircle/Models/Action/Action.swift:16]: name here is Action name! Please disregard. Looks great!

File: FullCircle/FullCircle/Controllers/ActionListViewController/ActionListViewController.swift [hunk 2]
```diff
@@ -80,7 +80,7 @@ class ActionListViewController: UIViewController {
     }()
     
     //MARK: - Internal Properties
-    var actions = [Action]() {
+    private var actions = [Action]() {
```
Comment (msystang) [inline FullCircle/FullCircle/Controllers/ActionListViewController/ActionListViewController.swift]: Should remove private protection level here so we can use `actions` in the ActionListViewController extensions where the tableView delegate and datasource functions are @davidlawrencer

Comment (davidlawrencer) [reply FullCircle/FullCircle/Controllers/ActionListViewController/ActionListViewController.swift]: fixed @msystang

**Review by msystang — APPROVED** [review]

**Pull request merged**